pkgs/borgbackup/borg/pkg.yaml

```diff
@@ -1,22 +1,2 @@
 packages:
-  - name: borgbackup/borg@1.1.12
-  - name: borgbackup/borg
-    version: 1.1.0b2
-  - name: borgbackup/borg
-    version: 1.0.10
-  - name: borgbackup/borg
-    version: 1.0.3
-  - name: borgbackup/borg
-    version: 1.0.2
-  - name: borgbackup/borg
-    version: 1.0.1
-  - name: borgbackup/borg
-    version: 1.0.0
-  - name: borgbackup/borg
-    version: 0.30.1
-  - name: borgbackup/borg
-    version: 0.30.0
-  - name: borgbackup/borg
-    version: 0.28.2
-  - name: borgbackup/borg
```

Comment:
aqua-registry should support not only the latest version but also old versions as much as possible.
If some versions are so old that they don't work on recent machine, we can exclude them, but otherwise we should support them as much as possible.

Reply:
> FWIW: Scaffolding added a whole bunch of old versions and accompanying overrides. Instead of digging through years-old releases to fix all the overrides, I went ahead and deleted the old versions. Hope that doesn't break assumptions. 😅

Sorry. I missed your comment. I'm checking.

Reply:
I reverted 3730a39eeddeb3f32fe19efd3569cd15cc62c217 and I simplified version_overrides while supporting old versions. 429bf479f3423baebe2e6c95e83d545c97131f67

Reply:
Thanks for the cleanup, and sorry for the mess. 🙏

TBH, I couldn't make sense of the list of versions the scaffolding chose to include. 😅

>  but also old versions as much as possible

I suspect there's diminishing returns the farther back you go -- and potential security issues, too. But then, responsibility of _using_ old versions is, ultimately, on the user.

Reply:
> I suspect there's diminishing returns the farther back you go -- and potential security issues, too. But then, responsibility of using old versions is, ultimately, on the user.

Exactly. We leave the responsibility to users.
aqua-registry supports all versions as much as possible.
If we feel it's too hard to support old versions, we may give up supporting them.